### PR TITLE
Past project code review

### PR DESCRIPTION
**Pull request by ifzyy**
### Being a code reviewer

- check if correct Github workflow is used
- check linters configuration
- check if correct HTML tags were used

### CODE REVIEW

**Comment by ifzyy** [comment]
Good job so far I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on.
143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91
 ### To highlight

- page looks good

**Comment by ifzyy** [comment]
Screenshot 2022-05-30 111542
 i think you should use ul instead of ol e.g
`<ul><li></li></ul>`

**Comment by ifzyy** [comment]
Screenshot 2022-05-30 111628
I think you should use the header tag instead of the div

**Comment by ifzyy** [comment]
I think you should set the button attribute "type". The value of the "type" attribute tells the browser to provide a regular button that has no associated action.
Screenshot 2022-05-30 225808

**Comment by ifzyy** [comment]
Your commit message should be more professional i.e instead of just index.html you can use "add index.html" 
Screenshot 2022-05-30 225705

**Comment by ifzyy** [comment]
Screenshot 2022-05-30 230344
It would help if you only had index.html as your only HTML file

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @ifzyy  ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Great job reviewing yourself! You found a lot of things to improve upon which goes to show the improvements you've made in your time at Microverse. Keep it up! 

## Required Changes ♻️

- [ ] This project is designed to get you to perform a review as if you were a code reviewer. The use of inline comments is paramount for this task. I recommend you take a closer look at [this documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request) to have a better idea of what this lesson wants you to learn. While your comments were accurate, you didn't make proper use of inline comments. Instead of taking all of those screenshots, you could've used inline comments. Kindly fix that.

_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: landingpage.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        <div class="smallquality">
+            Quality Assurance
+            For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument.</div>
+    </div>
+
+</div>
+<div class="contain">
+    <div class="box">
+        <div class="graybox">TENOR TROMBONE</div>
+        <ol>
```
Comment (ifzyy) [inline landingpage.html:54]: Use the ul instead of the ol i.e`<ul><li></li></ul>`

File: landingpage.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            <li> <b>$800</b></li>
+            <li>Plays similar to a Trumpet</li>
+            <li> Great for Jazz Bands</li>
+            <li>Lorem ipsum dolor.</li> 
+            <li>Lorem ipsum.</li> 
+        </ol>
+
+        <button>SELECT</button>
```
Comment (ifzyy) [inline landingpage.html:62]: I think you should set the button attribute "type". The value of the "type" attribute tells the browser to provide a regular button that has no associated action.

File: landingpage.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Product Landing page</title>
+    <link rel="stylesheet" href="css/style.css">
+
+</head>
+<body>
+<div class="header">
```
Comment (ifzyy) [inline landingpage.html:12]: I think you should use the header tag instead of the div

**Review by zhadier — APPROVED** [review]
## Hi @ifzyy  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

### Keep Rocking!👏👏👏
## Optional
- [ ] [_Optional_] Please make sure you add a why for all your requirements. The reviewee must understand why he is required to make a change.
<hr/>

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**